common/services/prismic/pages.js

```diff
@@ -195,6 +199,7 @@ export async function getPageSiblings(
       id: page.parentPages[i].id,
       title: page.parentPages[i].title,
       siblings: results.results.filter(sibling => sibling.id !== page.id),
+      collection: results.results,
```

Comment:
Do we use this anywhere?

Reply:
Don't think so, no. A hangover from our initial pairing perhaps. I'll get rid.